### ISSUE TITLE
Support for Musicscore's 2.2 sf3 format

### ISSUE THREAD

**Issue by light2yellow** [FluidSynth/fluidsynth#381]
FluidSynth version: 1.1.10

Not sure if I do everything correctly, but I cannot load the default sf3 from Musicscore 2.2 -- looks like something has changed:
```
> load /usr/share/mscore-2.2/sound/MuseScore_General.sf3 
fluidsynth: warning: Sample 'Pop D1' start/end file positions are invalid, disabling and will not be saved
fluidsynth: error: File contains data in an unknown format.
fluidsynth: error: Failed to load SoundFont "/usr/share/mscore-2.2/sound/MuseScore_General.sf3"
failed to load the SoundFont
```

It can be obtained here: ftp://ftp.osuosl.org/pub/musescore/soundfont/MuseScore_General/MuseScore_General.sf3

General sf3 support ticket is #140.

**Comment by mawe42** [comment]
SF3 support was broken in 1.1.10 and has been fixed in 1.1.11. Please check out the new version and try again.

**Comment by light2yellow** [comment]
Thanks!
